app/javascript/components/places/place_show.jsx

```diff
@@ -56,7 +56,7 @@ class PlaceShow extends React.Component {
             </Grid>
             <Grid container>
               {
-                place.events.map((event, i) =>
+                events.map((event, i) =>
```

Comment:
Try `({event: {name, address, start_time, end_time, first_imge_url}, i})` to get rid of repetition of `event`. Then use it wisely on line 61-62 (`name={name}`)